backend/server/routers/model.py

```diff
@@ -99,9 +99,11 @@ class PlannerData(BaseModel):
     mostRecentPastTerm: dict
 
 
-class Courses (BaseModel):
+class Courses(BaseModel):
     courses: list[str]
 
+class GenEdCourses(BaseModel):
+    courses: dict[str, str] = {}
```

Comment:
Is this model here to just differentiate between courses being returned as just a list of coursecodes vs dict with code and title?
If so, it might be good to decouple this from geneds specifically and just have a general type where the `dict` option is actually `Courses` and the list option is something like `CourseList` or `CourseCodes`.